packages/@adobe/spectrum-css-temp/components/table/index.css

```diff
@@ -150,6 +150,10 @@ governing permissions and limitations under the License.
 }
 
 .spectrum-Table-cellContents {
+  display: flex;
+  justify-content: inherit;
+  flex: 1 1 0%;
```

Comment:
we'll need a `min-width: 0px;` so that it can actually collapse I think
try making a cell where the contents are bigger than the available space

Reply:
Hmm, not sure I see the difference here with the min-width applied, but it is def weird that it is extending so far
![image](https://user-images.githubusercontent.com/9661127/122833588-14c63100-d2a2-11eb-8d85-9bfd2cde8e79.png)